### ISSUE TITLE
Pregel Bootstrap issue with build.gradle file dependencies

### ISSUE THREAD

**Issue by adamggibbs** [neo4j/graph-data-science#161]
**Describe the bug**
I'm trying to use the Pregel API and I downloaded the pregel bootstrap example along with its gradle build file from this repo into my IDE (IntelliJ Idea community). When gathering dependencies, I keep getting the following error: 

"Could not find com.neo4j.gds:neo4j-kernel-adapter-4.3.0-drop05.0:2.0.0-alpha01" 

I hadn't altered the files in any way so I expected all dependencies to be available and cause no errors. 

I'm wondering if there is any issue in the fact it keeps looking for "com.neo4j.gds: ..." rather than "org.neo4j.gds: ..." like all the other dependencies. For example, I can find the .jar files at https://repo1.maven.org/maven2/org/neo4j/gds/ but it seems like maven is only looking to https://repo1.maven.org/maven2/com/neo4j/gds/ which is throwing errors. Also in this [pom.xml](https://repo1.maven.org/maven2/org/neo4j/gds/alpha-algo/2.0.0-alpha02/alpha-algo-2.0.0-alpha02.pom) file from a neo4j GDS repo on maven central, the dependency uses com as well.

**Comment by DarthMax** [comment]
Hej @adamggibbs,

It seems that you are using the Pregel boostrap project from the current master branch. The master branch is what we use for developing the next version of GDS. This is also means that the Pregel bootstrap example points to a GDS version that is not release yet.
There are two ways you could fix your problem:
1. Switch to the 1.8 branch and use the Pregel bootstrap example from there: https://github.com/neo4j/graph-data-science/tree/1.8/examples/pregel-bootstrap
2. Keep your already set up project and change the GDS version to 1.8.4 as shown here: https://github.com/neo4j/graph-data-science/blob/1.8/examples/pregel-bootstrap/build.gradle#L10

Please let us know if you encounter further problems.
Best, Max

**Comment by adamggibbs** [comment]
Yes, switching to version 1.8.2 solved all the problems. Thank you!
